checkov/common/checks_infra/solvers/attribute_solvers/length_greater_than_attribute_solver.py

```diff
@@ -0,0 +1,28 @@
+from typing import List, Optional, Any, Dict
+from collections.abc import Sized
+from checkov.common.checks_infra.solvers.attribute_solvers.base_attribute_solver import BaseAttributeSolver
+from checkov.common.graph.checks_infra.enums import Operators
+from checkov.common.util.type_forcers import force_int
+
+
+class LengthGreaterThanAttributeSolver(BaseAttributeSolver):
+    operator = Operators.LENGTH_GREATER_THAN  # noqa: CCE003  # a static attribute
+
+    def _get_operation(self, vertex: Dict[str, Any], attribute: Optional[str]) -> bool:  # type:ignore[override]
+        if vertex.get(attribute) is None:  # type:ignore[arg-type]  # due to attribute can be None
+            return False
+
+        attr = vertex.get(attribute)  # type:ignore[arg-type]  # due to attribute can be None
```

Comment:
```suggestion
        attr = vertex.get(attribute)  # type:ignore[arg-type]  # due to attribute can be None
        if attr is None:  # type:ignore[arg-type]  # due to attribute can be None
            return False
        
```
you can just move it up and then you don't need to call it twice 🙂